.changeset/four-states-open.md

```diff
@@ -0,0 +1,5 @@
+---
+"@sumup-oss/circuit-ui": patch
+---
+
+Change plan details on ComparisonTable docs
```

Comment:
```suggestion
Updated the examples in the ComparisonTable documentation to be more generic.
```

Reply:
Actually, @sirineJ has a good point that a changeset isn't needed since this is a docs-only change.